pages/api/users/[username].js

```diff
@@ -167,6 +167,45 @@ export default async function handler(req, res) {
   const latestProfile = await Profile.findOne({ username });
   const links = await Link.find({ profile: latestProfile._id });
 
+  if (data.projects?.github) {
+    data.projects.github = await Promise.all(
+      data.projects.github.map(async (repo) => {
+        try {
+          const res = await fetch(`https://api.github.com/repos/${repo}`);
```

Comment:
We will hit the GitHub rate limit very quickly, we need to think how to cache these

Reply:
Don't have any prior experience in catching api calls but will look into this.